src/Dependencies/Collections/Microsoft.CodeAnalysis.Collections.shproj

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project>
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>e919dd77-34f8-4f57-8058-4d3ff4c2b241</ProjectGuid>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
```

Comment:
Do we need this property anymore?

Reply:
I'm not sure. I just copied it from other shared projects we have.